src/components/frame/v5/pages/AgreementsPage/AgreementsPage.tsx

```diff
@@ -0,0 +1,129 @@
+import { motion } from 'framer-motion';
+import React, { type FC } from 'react';
+
+import { ACTION } from '~constants/actions.ts';
+import { useActionSidebarContext } from '~context/ActionSidebarContext/index.tsx';
+import { useColonyContext } from '~context/ColonyContext.tsx';
+import {
+  useSetPageBreadcrumbs,
+  useSetPageHeadingTitle,
+} from '~context/PageHeadingContext/hooks.ts';
+import { useGetColonyActionsQuery, ColonyActionType } from '~gql';
+import { useCreateTeamBreadcrumbs } from '~hooks/useTeamsBreadcrumbs.ts';
+import { notNull } from '~utils/arrays/index.ts';
+import { formatText } from '~utils/intl.ts';
+import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts.tsx';
+import EmptyContent from '~v5/common/EmptyContent/EmptyContent.tsx';
+import Button from '~v5/shared/Button/Button.tsx';
+
+import AgreementCard from './partials/AgreementCard/index.ts';
+import AgreementCardSkeleton from './partials/AgreementCardSkeleton.tsx';
+
+const displayName = 'v5.pages.AgerementsPage';
+
+const AgreementsPage: FC = () => {
+  const {
+    colony: { colonyAddress },
+  } = useColonyContext();
+  const teamsBreadcrumbs = useCreateTeamBreadcrumbs();
+  const {
+    actionSidebarToggle: [, { toggleOn: toggleActionSidebarOn }],
+  } = useActionSidebarContext();
+
+  useSetPageBreadcrumbs(teamsBreadcrumbs);
+  useSetPageHeadingTitle(formatText({ id: 'agreementsPage.title' }));
+
+  const { data: agreementsData, loading } = useGetColonyActionsQuery({
+    variables: {
+      colonyAddress,
+      filter: {
+        showInActionsList: { eq: false },
```

Comment:
What is the purpose of this filter? It will hide any agreements that have met the 10% staking threshold.